compiler/safety/run.ml

```diff
@@ -31,11 +31,13 @@ let load_file name =
   |> Compile.preprocess Arch.reg_size Arch.asmOp
 
 let analyze ~fmt pd asmOp source_f_decl f_decl p =
-  let module AbsInt = SafetyInterpreter.AbsAnalyzer (struct
+  let module PW = struct
+    type extended_op = X86_extra.x86_extended_op
```

Comment:
`pd` and `asmOp` should be in the functor. `PW`, on the contrary, should be normal function arguments.